cmd/ping/main.go

```diff
@@ -82,3 +90,28 @@ func main() {
 
 	log.Fatal(http.ListenAndServe(":"+port, nil))
 }
+
+// endpointsCache is used to fetch a value from the local cache if available
+// and from the Cloud Rur Admin API in case it is not available, or has expired.
+func endpointsCache(c *cache.Cache) map[string]config.Endpoint {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	em, found := c.Get("map")
+	if !found {
+		e, err := config.GenerateConfigFromAPI(ctx)
+		if err != nil {
+			em, found := c.Get("map_fallback")
```

Comment:
Could this be `stale_endpoints` or `static_endpoints`? A little comment here might help too, e.g., "Retrieve stale endpoints when the fresh endpoints are unavailable."